src/app/actions.ts

```diff
@@ -8,12 +8,16 @@ export async function autocomplete(query: string) {
 	if (!query) {
 		return [];
 	}
-	let dsps = await prisma.domainSelectorPair.findMany({
-		distinct: ['domain'],
-		where: { domain: { startsWith: query } },
-		orderBy: { domain: 'asc' },
-		take: 8
-	});
+	const modifiedQuery = query.replace(/\./g, "-");
```

Comment:
this is only in the autocomplete -- we want it on the whole page

Reply:
i.e. even clicking enter on a search for 'pse.dev' should pop up all the domains that have 'pse-dev' in the name.

Reply:
For instance right now even search for that gives us 'No records found for "pse-dev"' which is bad. We maybe even want to match something like catpse-dev.2021.google.com or pse-dev-staging.2021.google.com, but make sure that such a result is below the one that starts with pse-dev.2021.google.com or whatever (these arent real urls but ive seen this pattern).